src/components/button/button.css

```diff
@@ -0,0 +1,67 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.interactivity {
+  cursor: pointer;
+}
+
+.typography {
+  font-family: var(--mavenlink-type-font-family);
+  font-size: var(--mavenlink-type-button-font-size);
+  line-height: var(--spacing-x-large);
+}
+
+.button {
+  composes: typography;
+  composes: interactivity;
+  border-width: 1px;
+  border-radius: 3px;
+  border-style: solid;
+  padding-bottom: 0;
+  padding-left: var(--spacing-x-large);
+  padding-right: var(--spacing-x-large);
+  padding-top: 0;
+}
+
+.button:hover {
+  opacity: .7;
+}
+
+.button[disabled] {
+  background: #eee;
+  border-color: #ddd;
+  text-shadow: 0 1px 1px var(--white);
+  color: #ccc;
+  cursor: default !important;
+}
+
+.button[disabled]:hover {
+  opacity: 1;
+}
+
+.blank {
```

Comment:
Question for design: is there a concept for a disabled state on a blank button?

Reply:
Question for design: is there a "active" state for the blank button? legacy styleguide has some inset styling.